.github/workflows/github-ci.yml

```
@@ -0,0 +1,126 @@
+  build:
+    name: Build Front Application
+    runs-on: ubuntu-latest
+    needs: [rebase-check]
+    if: |
+      (github.event_name == 'pull_request' && success()) ||
+      (github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop')) ||
+      (github.event_name == 'pull_request_target' && github.event.pull_request.merged == true)
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)